src/openeo_grass_gis_driver/actinia_processing/base.py

```diff
@@ -63,6 +63,7 @@ class GrassDataType(Enum):
     VECTOR = "vector"
     STRDS = "strds"
     STAC = "rastercube"
+    LOCAL = "gdallocal"
```

Comment:
See below for naming suggestions. Would it be bad for parsing if it would be the same as for STAC due to similar format?
```suggestion
    LOCAL = "rastercube"
```

Reply:
See comment above, STAC and LOCAL must be different datatypes, otherwise the `is_stac()` method no longer works.

Reply:
STAC and local collection now have the same datatype "rastercube", but different datasources such that we can distinguish between these (and GRASS native collections).

Reply:
:+1: